crm_project_task/tests/test_crm_project_task.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+from odoo.tests import tagged
+
+from .common import CrmProjectTaskCommon
+
+
+@tagged("post_install", "-at_install")
+class TestCrmProjectTask(CrmProjectTaskCommon):
+    def setUp(self):
+        super().setUp()
+
+    def test_create_task(self):
+        self.task_name = "Test"
+        self.task_description = "Line1</br>Line2"
+        prev_tasks = self.project.task_ids
+        self.env["crm.create.task"].create(
```

Comment:
According to https://github.com/OCA/crm/issues/481:

> Salesman could create a task in an existing project without having permissions in project app

So I think this test would benefit if being executed by a user with only salesman permissions. Then, a green test would be checking that behavior too.

Reply:
Again you're right